app/views/comments/_comment.html.erb

```diff
@@ -25,6 +25,7 @@
       "></div>
 <div id="c_<%= comment.short_id %>"
   data-shortid="<%= comment.short_id if comment.persisted? %>"
+  data-id="<%= comment.id %>"
```

Comment:
```suggestion
```

Don't want to leak PKs, they become a weird status symbol.

Reply:
sure deleted id and used shortid